### ISSUE TITLE
Diablo IV  - Error - No GPUs found

### ISSUE THREAD

**Issue by LORDSANSUI** [lutris/lutris#4774]
### Bug description

Hello,

Currently unable to play Diablo IV due to Lutris/Wine error "No GPUs found" so the game can't launch.

The game is not native available for Linux nor Linux is supported from Blizzard developers, so thanks in advanced if the community could make it work as Diablo 3 and Starcraft II, both are currently installed on my computer and run flawlessly.

### How to Reproduce

1. Open Lutris
2. Launch Battle.net
3. Select the game Diablo IV Beta
4. Click on Pay button.

### Expected behavior

I was expecting the game to launch and run as others Blizzard titles.

### Log output

```shell
INFO     2023-03-17 15:34:41,717 [startup.init_lutris:173]:Starting Lutris 0.5.12
DEBUG    2023-03-17 15:34:41,773 [xrandr._get_vidmodes:16]:Retrieving video modes from XrandR
INFO     2023-03-17 15:34:41,811 [startup.check_driver:65]:Running AMD Mesa driver 22.3.5 on AMD Radeon RX 570 Series (polaris10, LLVM 15.0.7, DRM 3.49, 6.2.6-1-MANJARO) (0x67df)
INFO     2023-03-17 15:34:41,811 [startup.check_driver:77]:GPU: 1002:67DF 1462:341B (amdgpu drivers)
DEBUG    2023-03-17 15:34:42,002 [lutriswindow.update_store:448]:Showing 1 games
INFO     2023-03-17 15:34:42,011 [startup.update_runtimes:218]:Startup complete
ERROR    2023-03-17 15:34:47,233 [nvidia.get_nvidia_glx_path:42]:Unable to load libGLX_nvidia.so.0
WARNING  2023-03-17 15:34:47,233 [nvidia.get_nvidia_dll_path:90]:Unable to locate libGLX_nvidia
DEBUG    2023-03-17 15:34:47,343 [xrandr._get_vidmodes:16]:Retrieving video modes from XrandR
DEBUG    2023-03-17 15:34:47,349 [command.start:141]:DISABLE_LAYER_AMD_SWITCHABLE_GRAPHICS_1="1"
DEBUG    2023-03-17 15:34:47,349 [command.start:141]:__GL_SHADER_DISK_CACHE="1"
DEBUG    2023-03-17 15:34:47,349 [command.start:141]:__GL_SHADER_DISK_CACHE_PATH="/home/lordsansui/GAMES/battlenet"
DEBUG    2023-03-17 15:34:47,350 [command.start:141]:VK_ICD_FILENAMES="/usr/share/vulkan/icd.d/radeon_icd.x86_64.json:/usr/share/vulkan/icd.d/radeon_icd.i686.json"
DEBUG    2023-03-17 15:34:47,350 [command.start:141]:LD_LIBRARY_PATH="/home/lordsansui/.local/share/lutris/runners/wine/lutris-GE-Proton7-20-x86_64/lib:/home/lordsansui/.local/share/lutris/runners/wine/lutris-GE-Proton7-20-x86_64/lib64:/usr/lib:/usr/lib32:/usr/lib/libfakeroot:/usr/lib64:/home/lordsansui/.local/share/lutris/runtime/Ubuntu-18.04-i686:/home/lordsansui/.local/share/lutris/runtime/steam/i386/lib/i386-linux-gnu:/home/lordsansui/.local/share/lutris/runtime/steam/i386/lib:/home/lordsansui/.local/share/lutris/runtime/steam/i386/usr/lib/i386-linux-gnu:/home/lordsansui/.local/share/lutris/runtime/steam/i386/usr/lib:/home/lordsansui/.local/share/lutris/runtime/Ubuntu-18.04-x86_64:/home/lordsansui/.local/share/lutris/runtime/steam/amd64/lib/x86_64-linux-gnu:/home/lordsansui/.local/share/lutris/runtime/steam/amd64/lib:/home/lordsansui/.local/share/lutris/runtime/steam/amd64/usr/lib/x86_64-linux-gnu:/home/lordsansui/.local/share/lutris/runtime/steam/amd64/usr/lib"
DEBUG    2023-03-17 15:34:47,350 [command.start:141]:=""
DEBUG    2023-03-17 15:34:47,350 [command.start:141]:WINEDEBUG="-all"
DEBUG    2023-03-17 15:34:47,350 [command.start:141]:DXVK_LOG_LEVEL="none"
DEBUG    2023-03-17 15:34:47,350 [command.start:141]:WINEARCH="win64"
DEBUG    2023-03-17 15:34:47,350 [command.start:141]:WINE="/home/lordsansui/.local/share/lutris/runners/wine/lutris-GE-Proton7-20-x86_64/bin/wine"
DEBUG    2023-03-17 15:34:47,350 [command.start:141]:WINE_MONO_CACHE_DIR="/home/lordsansui/.local/share/lutris/runners/wine/lutris-GE-Proton7-20-x86_64/mono"
DEBUG    2023-03-17 15:34:47,350 [command.start:141]:WINE_GECKO_CACHE_DIR="/home/lordsansui/.local/share/lutris/runners/wine/lutris-GE-Proton7-20-x86_64/gecko"
DEBUG    2023-03-17 15:34:47,350 [command.start:141]:GST_PLUGIN_SYSTEM_PATH_1_0="/home/lordsansui/.local/share/lutris/runners/wine/lutris-GE-Proton7-20-x86_64/lib64/gstreamer-1.0/:/home/lordsansui/.local/share/lutris/runners/wine/lutris-GE-Proton7-20-x86_64/lib/gstreamer-1.0/"
DEBUG    2023-03-17 15:34:47,351 [command.start:141]:WINEPREFIX="/home/lordsansui/GAMES/battlenet/battlenet"
DEBUG    2023-03-17 15:34:47,351 [command.start:141]:WINEESYNC="1"
DEBUG    2023-03-17 15:34:47,351 [command.start:141]:WINEFSYNC="1"
DEBUG    2023-03-17 15:34:47,351 [command.start:141]:WINE_FULLSCREEN_FSR="1"
DEBUG    2023-03-17 15:34:47,351 [command.start:141]:DXVK_NVAPIHACK="0"
DEBUG    2023-03-17 15:34:47,351 [command.start:141]:DXVK_ENABLE_NVAPI="1"
DEBUG    2023-03-17 15:34:47,351 [command.start:141]:PROTON_BATTLEYE_RUNTIME="/home/lordsansui/.local/share/lutris/runtime/battleye_runtime"
DEBUG    2023-03-17 15:34:47,351 [command.start:141]:PROTON_EAC_RUNTIME="/home/lordsansui/.local/share/lutris/runtime/eac_runtime"
DEBUG    2023-03-17 15:34:47,351 [command.start:141]:WINEDLLOVERRIDES="d3d10core,d3d11,d3d12,d3d9,d3dcompiler_33,d3dcompiler_34,d3dcompiler_35,d3dcompiler_36,d3dcompiler_37,d3dcompiler_38,d3dcompiler_39,d3dcompiler_40,d3dcompiler_41,d3dcompiler_42,d3dcompiler_43,d3dcompiler_46,d3dcompiler_47,d3dx10,d3dx10_33,d3dx10_34,d3dx10_35,d3dx10_36,d3dx10_37,d3dx10_38,d3dx10_39,d3dx10_40,d3dx10_41,d3dx10_42,d3dx10_43,d3dx11_42,d3dx11_43,d3dx9_24,d3dx9_25,d3dx9_26,d3dx9_27,d3dx9_28,d3dx9_29,d3dx9_30,d3dx9_31,d3dx9_32,d3dx9_33,d3dx9_34,d3dx9_35,d3dx9_36,d3dx9_37,d3dx9_38,d3dx9_39,d3dx9_40,d3dx9_41,d3dx9_42,d3dx9_43,dxgi,nvapi,nvapi64,nvml=n;winemenubuilder="
DEBUG    2023-03-17 15:34:47,351 [command.start:141]:WINE_LARGE_ADDRESS_AWARE="1"
DEBUG    2023-03-17 15:34:47,352 [command.start:141]:game_name="Blizzard Battle.net"
DEBUG    2023-03-17 15:34:47,352 [command.start:141]:PYTHONPATH="/usr/lib/lutris:/usr/bin:/usr/lib/python310.zip:/usr/lib/python3.10:/usr/lib/python3.10/lib-dynload:/usr/lib/python3.10/site-packages"
DEBUG    2023-03-17 15:34:47,352 [command.start:141]:LUTRIS_GAME_UUID="d753e4da-751b-41d2-a193-ac38320be7d6"
Discord ID: None
INFO     2023-03-17 15:34:47,373 [game.start_game:586]:Discord RPC Disabled or Discord APP ID Not Present
lutris-wrapper: Blizzard Battle.net
Started initial process 10607 from /home/lordsansui/.local/share/lutris/runners/wine/lutris-GE-Proton7-20-x86_64/bin/wine /home/lordsansui/GAMES/battlenet/drive_c/Program Files (x86)/Battle.net/Battle.net.exe
Start monitoring process.
fsync: up and running.
wine: RLIMIT_NICE is <= 20, unable to use setpriority safely
[0317/153451.560:ERROR:network_change_notifier_win.cc(142)] WSALookupServiceBegin failed with: 0
[0317/153452.325:ERROR:dxva_video_decode_accelerator_win.cc(1397)] DXVAVDA fatal error: could not LoadLibrary: msmpeg2vdec.dll: Module not found. (0x7E)
[0317/153452.326:ERROR:network_change_notifier_win.cc(142)] WSALookupServiceBegin failed with: 0
[0317/153452.327:ERROR:dxva_video_decode_accelerator_win.cc(1405)] DXVAVDA fatal error: could not LoadLibrary: msvproc.dll: Module not found. (0x7E)
[0317/153452.486:ERROR:dxva_video_decode_accelerator_win.cc(1320)] DXVAVDA fatal error: Could not load msmpeg2vdec.dll: Module not found. (0x7E)
[0317/153537.046:ERROR:mf_audio_decoder.cc(418)] MSAudDecMFT.dll nor Msmpeg2adec.dll is not loaded. Load it in IO thread. (Not here.)
[0317/153537.047:ERROR:mf_audio_decoder.cc(179)] Failed to find audio decoder: -2147467259
[0317/153537.047:ERROR:mf_audio_decoder.cc(709)] Failed to initialize MfAudioDecoder
[0317/153537.047:ERROR:batching_media_log.cc(38)] MediaEvent: {"error":"audio decoder initialization failed"}
[0317/153537.047:ERROR:batching_media_log.cc(35)] MediaEvent: {"pipeline_error":15}
[0317/153652.534:ERROR:dxva_video_decode_accelerator_win.cc(1397)] DXVAVDA fatal error: could not LoadLibrary: msmpeg2vdec.dll: Module not found. (0x7E)
[0317/153652.537:ERROR:dxva_video_decode_accelerator_win.cc(1405)] DXVAVDA fatal error: could not LoadLibrary: msvproc.dll: Module not found. (0x7E)
ERROR    2023-03-17 15:42:31,829 [process.environ:121]:Failed to parse environment variables: 75,131072
ERROR    2023-03-17 15:42:33,833 [process.environ:121]:Failed to parse environment variables: 75,131072
ERROR    2023-03-17 15:42:35,833 [process.environ:121]:Failed to parse environment variables: 75,131072
[0317/154413.019:INFO:cef_logging.h(628)] BNET, sending query, close window
Initial process has exited (return code: 0)
Monitored process exited.
Exit with return code 0
DEBUG    2023-03-17 15:44:33,062 [command.on_stop:195]:Process 10605 has terminated with code 0
DEBUG    2023-03-17 15:44:33,941 [game.beat:693]:Game thread stopped
WARNING  2023-03-17 15:44:33,942 [game.on_game_quit:730]:Game still running (state: running)
INFO     2023-03-17 15:44:33,942 [game.stop:704]:Stopping Blizzard Battle.net (wine)
DEBUG    2023-03-17 15:44:33,943 [game.stop_game:664]:Blizzard Battle.net (wine) has run for 586 seconds
DEBUG    2023-03-17 15:44:33,976 [game.on_game_quit:748]:Blizzard Battle.net stopped at sex, 17 mar 2023 15:44:33
DEBUG    2023-03-17 15:44:33,977 [game.save:299]:Saving Blizzard Battle.net (wine) with config ID blizzard-battlenet-standard-1657326946
```

### System Information

```shell
System:
  Kernel: 6.2.6-1-MANJARO arch: x86_64 bits: 64 compiler: gcc v: 12.2.1
    parameters: BOOT_IMAGE=/@/boot/vmlinuz-6.2-x86_64
    root=UUID=69678a39-1940-48ee-8d02-88e765985137 rw rootflags=subvol=@ quiet
    apparmor=1 security=apparmor
    resume=UUID=b136d668-4460-4cf7-9211-93b9a5283ab3 udev.log_priority=3
  Desktop: KDE Plasma v: 5.26.5 tk: Qt v: 5.15.8 wm: kwin_wayland vt: 2
    dm: SDDM Distro: Manjaro Linux base: Arch Linux
Machine:
  Type: Desktop Mobo: N/A model: X99 serial: <superuser required>
    UEFI: American Megatrends v: 5.11 date: 07/30/2021
Battery:
  Message: No system battery data found. Is one present?
Memory:
  RAM: total: 15.53 GiB used: 5.51 GiB (35.5%)
  RAM Report: permissions: Unable to run dmidecode. Root privileges required.
CPU:
  Info: model: Intel Xeon E5-2640 v3 bits: 64 type: MT MCP arch: Haswell
    level: v3 note: check built: 2013-15 process: Intel 22nm family: 6
    model-id: 0x3F (63) stepping: 2 microcode: 0x49
  Topology: cpus: 1x cores: 8 tpc: 2 threads: 16 smt: enabled cache:
    L1: 512 KiB desc: d-8x32 KiB; i-8x32 KiB L2: 2 MiB desc: 8x256 KiB L3: 20 MiB
    desc: 1x20 MiB
  Speed (MHz): avg: 1569 high: 3400 min/max: 1200/3400 scaling:
    driver: intel_cpufreq governor: schedutil cores: 1: 3400 2: 1197 3: 1197
    4: 1200 5: 1197 6: 1197 7: 3018 8: 1197 9: 1357 10: 1197 11: 1197 12: 1200
    13: 1199 14: 1197 15: 2955 16: 1200 bogomips: 83043
  Flags: abm acpi aes aperfmperf apic arat arch_perfmon avx avx2 bmi1 bmi2
    bts clflush cmov constant_tsc cpuid cpuid_fault cqm cqm_llc cqm_occup_llc
    cx16 cx8 dca de ds_cpl dtes64 dtherm dts epb ept ept_ad erms est f16c
    flexpriority flush_l1d fma fpu fsgsbase fxsr ht ibpb ibrs ida intel_ppin
    invpcid invpcid_single lahf_lm lm mca mce md_clear mmx monitor movbe msr
    mtrr nonstop_tsc nopl nx pae pat pbe pcid pclmulqdq pdcm pdpe1gb pebs pge
    pln pni popcnt pse pse36 pti pts rdrand rdtscp rep_good sdbg sep smep smx
    ss ssbd sse sse2 sse4_1 sse4_2 ssse3 stibp syscall tm tm2 tpr_shadow tsc
    tsc_adjust tsc_deadline_timer vme vmx vnmi vpid x2apic xsave xsaveopt
    xtopology xtpr
  Vulnerabilities:
  Type: itlb_multihit status: KVM: VMX disabled
  Type: l1tf mitigation: PTE Inversion; VMX: conditional cache flushes, SMT
    vulnerable
  Type: mds mitigation: Clear CPU buffers; SMT vulnerable
  Type: meltdown mitigation: PTI
  Type: mmio_stale_data mitigation: Clear CPU buffers; SMT vulnerable
  Type: retbleed status: Not affected
  Type: spec_store_bypass mitigation: Speculative Store Bypass disabled via
    prctl
  Type: spectre_v1 mitigation: usercopy/swapgs barriers and __user pointer
    sanitization
  Type: spectre_v2 mitigation: Retpolines, IBPB: conditional, IBRS_FW,
    STIBP: conditional, RSB filling, PBRSB-eIBRS: Not affected
  Type: srbds status: Not affected
  Type: tsx_async_abort status: Not affected
Graphics:
  Device-1: AMD Ellesmere [Radeon RX 470/480/570/570X/580/580X/590]
    vendor: Micro-Star MSI driver: amdgpu v: kernel arch: GCN-4
    code: Arctic Islands process: GF 14nm built: 2016-20 pcie: gen: 3
    speed: 8 GT/s lanes: 16 ports: active: DP-3 empty: DP-1, DP-2, DVI-D-1,
    HDMI-A-1 bus-ID: 03:00.0 chip-ID: 1002:67df class-ID: 0300 temp: 54.0 C
  Display: wayland server: X.org v: 1.21.1.7 with: Xwayland v: 22.1.8
    compositor: kwin_wayland driver: X: loaded: amdgpu unloaded: modesetting
    alternate: fbdev,vesa dri: radeonsi gpu: amdgpu display-ID: 0
  Monitor-1: DP-3 res: 2560x1440 size: N/A modes: N/A
  API: OpenGL v: 4.6 Mesa 22.3.5 renderer: AMD Radeon RX 570 Series
    (polaris10 LLVM 15.0.7 DRM 3.49 6.2.6-1-MANJARO) direct-render: Yes
Audio:
  Device-1: Intel 8 Series/C220 Series High Definition Audio
    driver: snd_hda_intel v: kernel bus-ID: 00:1b.0 chip-ID: 8086:8c20
    class-ID: 0403
  Device-2: AMD Ellesmere HDMI Audio [Radeon RX 470/480 / 570/580/590]
    vendor: Micro-Star MSI driver: snd_hda_intel v: kernel pcie: gen: 3
    speed: 8 GT/s lanes: 16 bus-ID: 03:00.1 chip-ID: 1002:aaf0 class-ID: 0403
  Sound API: ALSA v: k6.2.6-1-MANJARO running: yes
  Sound Server-1: JACK v: 1.9.22 running: no
  Sound Server-2: PulseAudio v: 16.1 running: yes
  Sound Server-3: PipeWire v: 0.3.65 running: yes
Network:
  Device-1: Realtek RTL8111/8168/8411 PCI Express Gigabit Ethernet
    driver: r8169 v: kernel pcie: gen: 1 speed: 2.5 GT/s lanes: 1 port: d000
    bus-ID: 06:00.0 chip-ID: 10ec:8168 class-ID: 0200
  IF: enp6s0 state: up speed: 1000 Mbps duplex: full mac: <filter>
  IP v4: <filter> type: dynamic noprefixroute scope: global
    broadcast: <filter>
  IP v6: <filter> type: noprefixroute scope: global
  IP v6: <filter> type: noprefixroute scope: link
  IF-ID-1: virbr0 state: down mac: <filter>
  IP v4: <filter> scope: global broadcast: <filter>
  WAN IP: <filter>
Bluetooth:
  Message: No bluetooth data found.
Logical:
  Message: No logical block device data found.
RAID:
  Message: No RAID data found.
Drives:
  Local Storage: total: 2.73 TiB used: 1.78 TiB (65.3%)
  SMART Message: Unable to run smartctl. Root privileges required.
  ID-1: /dev/nvme0n1 maj-min: 259:2 vendor: Western Digital
    model: WD Blue SN570 500GB size: 465.76 GiB block-size: physical: 512 B
    logical: 512 B speed: 31.6 Gb/s lanes: 4 type: SSD serial: <filter>
    rev: 234100WD temp: 40.9 C scheme: GPT
  ID-2: /dev/nvme1n1 maj-min: 259:0 vendor: Western Digital
    model: WDS500G2B0C-00PXH0 size: 465.76 GiB block-size: physical: 512 B
    logical: 512 B speed: 31.6 Gb/s lanes: 4 type: SSD serial: <filter>
    rev: 233010WD temp: 34.9 C scheme: GPT
  ID-3: /dev/sda maj-min: 8:0 vendor: Western Digital model: WD20PURZ-85GU6Y0
    size: 1.82 TiB block-size: physical: 4096 B logical: 512 B speed: 6.0 Gb/s
    type: HDD rpm: 5400 serial: <filter> rev: 0A80 scheme: MBR
  Message: No optical or floppy data found.
Partition:
  ID-1: / raw-size: 448.38 GiB size: 448.38 GiB (100.00%)
    used: 197.41 GiB (44.0%) fs: btrfs dev: /dev/nvme0n1p2 maj-min: 259:4
    label: LINUX uuid: 69678a39-1940-48ee-8d02-88e765985137
  ID-2: /boot/efi raw-size: 300 MiB size: 299.4 MiB (99.80%)
    used: 568 KiB (0.2%) fs: vfat dev: /dev/nvme0n1p1 maj-min: 259:3
    label: NO_LABEL uuid: CDC5-157E
  ID-3: /home raw-size: 448.38 GiB size: 448.38 GiB (100.00%)
    used: 197.41 GiB (44.0%) fs: btrfs dev: /dev/nvme0n1p2 maj-min: 259:4
    label: LINUX uuid: 69678a39-1940-48ee-8d02-88e765985137
  ID-4: /home/<filter>/GAMES raw-size: 465.76 GiB size: 457.38 GiB (98.20%)
    used: 240.77 GiB (52.6%) fs: ext4 dev: /dev/nvme1n1p1 maj-min: 259:1
    label: GAMES uuid: 9467047a-a0d0-46ff-859d-67a1bfff2456
  ID-5: /home/<filter>/HDD-2TB raw-size: 1.82 TiB size: 1.79 TiB (98.37%)
    used: 1.35 TiB (75.6%) fs: ext4 dev: /dev/sda1 maj-min: 8:1 label: 2TB-EXT4
    uuid: b9d4d0f9-d1ce-425d-8fe4-e2f830005289
  ID-6: /var/cache raw-size: 448.38 GiB size: 448.38 GiB (100.00%)
    used: 197.41 GiB (44.0%) fs: btrfs dev: /dev/nvme0n1p2 maj-min: 259:4
    label: LINUX uuid: 69678a39-1940-48ee-8d02-88e765985137
  ID-7: /var/log raw-size: 448.38 GiB size: 448.38 GiB (100.00%)
    used: 197.41 GiB (44.0%) fs: btrfs dev: /dev/nvme0n1p2 maj-min: 259:4
    label: LINUX uuid: 69678a39-1940-48ee-8d02-88e765985137
Swap:
  Kernel: swappiness: 10 (default 60) cache-pressure: 100 (default)
  ID-1: swap-1 type: partition size: 17.09 GiB used: 1.62 GiB (9.5%)
    priority: -2 dev: /dev/nvme0n1p3 maj-min: 259:5 label: swap
    uuid: b136d668-4460-4cf7-9211-93b9a5283ab3
Unmounted:
  Message: No unmounted partitions found.
USB:
  Hub-1: 1-0:1 info: Full speed or root hub ports: 2 rev: 2.0 speed: 480 Mb/s
    chip-ID: 1d6b:0002 class-ID: 0900
  Hub-2: 1-1:2 info: Intel Integrated Rate Matching Hub ports: 6 rev: 2.0
    speed: 480 Mb/s chip-ID: 8087:8008 class-ID: 0900
  Hub-3: 2-0:1 info: Full speed or root hub ports: 2 rev: 2.0 speed: 480 Mb/s
    chip-ID: 1d6b:0002 class-ID: 0900
  Hub-4: 2-1:2 info: Intel Integrated Rate Matching Hub ports: 6 rev: 2.0
    speed: 480 Mb/s chip-ID: 8087:8000 class-ID: 0900
  Hub-5: 3-0:1 info: Hi-speed hub with single TT ports: 12 rev: 2.0
    speed: 480 Mb/s chip-ID: 1d6b:0002 class-ID: 0900
  Device-1: 3-7:2 info: [Maxxter] USB GAMING MOUSE type: Mouse,Keyboard
    driver: hid-generic,usbhid interfaces: 2 rev: 1.1 speed: 1.5 Mb/s
    power: 100mA chip-ID: 18f8:0fc0 class-ID: 0301
  Device-2: 3-8:3 info: SINO WEALTH Gaming KB type: Keyboard,HID
    driver: hid-generic,usbhid interfaces: 2 rev: 1.1 speed: 12 Mb/s power: 500mA
    chip-ID: 258a:002a class-ID: 0300
  Hub-6: 4-0:1 info: Super-speed hub ports: 6 rev: 3.0 speed: 5 Gb/s
    chip-ID: 1d6b:0003 class-ID: 0900
Sensors:
  System Temperatures: cpu: 37.0 C mobo: N/A gpu: amdgpu temp: 54.0 C
  Fan Speeds (RPM): N/A gpu: amdgpu fan: 244
Info:
  Processes: 388 Uptime: 2h 14m wakeups: 0 Init: systemd v: 252
  default: graphical tool: systemctl Compilers: gcc: 12.2.1 clang: 15.0.7
  Packages: 1607 pm: pacman pkgs: 1567 libs: 446 tools: pamac pm: flatpak
  pkgs: 33 pm: snap pkgs: 7 Shell: Zsh v: 5.9 default: Bash v: 5.1.16
  running-in: konsole inxi: 3.3.25
```

### Media (optional)

![Screenshot_20230317_152702](https://user-images.githubusercontent.com/128180250/225990429-88da0690-3ffc-4e8e-8afe-380d61468cfb.png)

### Checklist:

- [X] I'm not asking for support with a game or the wine runner.
- [X] I have followed the above mentioned guides and have all the graphics and wine dependencies installed.
- [X] I have checked for existing issues that describe my problem prior to opening this one.
- [X] I understand that improperly formatted bug reports may be closed without explanation.

**Comment by selbyk** [comment]
I am also attempting to debug this issue. Except I am running an nvidia 3080ti. The same thing happened with D2R when it launched, iirc

**Comment by selbyk** [comment]
@LORDSANSUI there's a post with redditors having some luck: https://www.reddit.com/r/linux_gaming/comments/11tt2zd/for_those_wondering_about_diablo_4_beta_launching/?utm_source=share&utm_medium=android_app&utm_name=androidcss&utm_term=1&utm_content=share_button

**Comment by LORDSANSUI** [comment]
@selbyk thank you to share the link, I will study and see if I can replicate proposed solution.

**Comment by LORDSANSUI** [comment]
lutris-GE-Proton7-19 
dxvk-2.1
vkd3d-proton-2.8
+
Enable D3D Extras = true
Esync = true
Fsync = true
+
Environment Variables:
VKD3D_FEATURE_LEVEL=12_1

Above indication fixed the issue for me as suggested in the reddit link.

**Comment by cihlarma** [comment]
Related: #4777

**Comment by GloriousEggroll** [comment]
> Related: #4777

this is a separate issue and is not related to https://github.com/lutris/lutris/issues/4777. Closing this issue as it has been resolved with the wine-ge d4 build (and properly resolved in upstream proton)

**Comment by MTPeyton** [comment]
It appears this issue can still happen even with https://github.com/GloriousEggroll/proton-ge-custom/releases/tag/GE-Proton7-52

Not sure if something changed in the D4 open beta (as compared to the closed beta) to cause this reversion. Or maybe something is specifically wrong with my machine.

Logs
```
Started initial process 63785 from gamemoderun /home/mattgaming/Documents/Custom Wine Versions/GE-Proton7-52/files/bin/wine /home/mattgaming/Games/battlenet/drive_c/Program Files (x86)/Battle.net/Battle.net.exe
Start monitoring process.
ERROR: ld.so: object '/usr/$LIB/libgamemodeauto.so.0' from LD_PRELOAD cannot be preloaded (cannot open shared object file): ignored.
ERROR: ld.so: object '/usr/$LIB/libgamemodeauto.so.0' from LD_PRELOAD cannot be preloaded (cannot open shared object file): ignored.
esync: up and running.
wine: RLIMIT_NICE is <= 20, unable to use setpriority safely
ERROR: ld.so: object '/usr/$LIB/libgamemodeauto.so.0' from LD_PRELOAD cannot be preloaded (cannot open shared object file): ignored.
ERROR: ld.so: object '/usr/$LIB/libgamemodeauto.so.0' from LD_PRELOAD cannot be preloaded (cannot open shared object file): ignored.
wine: configuration in L"/home/mattgaming/Games/battlenet" has been updated.
ERROR: ld.so: object '/usr/$LIB/libgamemodeauto.so.0' from LD_PRELOAD cannot be preloaded (cannot open shared object file): ignored.
ERROR: ld.so: object '/usr/$LIB/libgamemodeauto.so.0' from LD_PRELOAD cannot be preloaded (cannot open shared object file): ignored.
[0324/174022.499:ERROR:network_change_notifier_win.cc(142)] WSALookupServiceBegin failed with: 0
ERROR: ld.so: object '/usr/$LIB/libgamemodeauto.so.0' from LD_PRELOAD cannot be preloaded (cannot open shared object file): ignored.
ERROR: ld.so: object '/usr/$LIB/libgamemodeauto.so.0' from LD_PRELOAD cannot be preloaded (cannot open shared object file): ignored.
[0324/174022.816:ERROR:network_change_notifier_win.cc(142)] WSALookupServiceBegin failed with: 0
[0324/174022.915:ERROR:dxva_video_decode_accelerator_win.cc(1405)] DXVAVDA fatal error: could not LoadLibrary: msvproc.dll: Module not found. (0x7E)
[0324/174023.103:ERROR:gl_surface_egl.cc(787)] EGL Driver message (Critical) eglInitialize: No available renderers.
[0324/174023.103:ERROR:gl_surface_egl.cc(1273)] eglInitialize D3D11 failed with error EGL_NOT_INITIALIZED, trying next display type
[0324/174023.207:ERROR:gl_surface_egl.cc(787)] EGL Driver message (Critical) eglInitialize: No available renderers.
[0324/174023.207:ERROR:gl_surface_egl.cc(1273)] eglInitialize D3D9 failed with error EGL_NOT_INITIALIZED
[0324/174023.207:ERROR:gl_initializer_win.cc(196)] GLSurfaceEGL::InitializeOneOff failed.
[0324/174023.214:ERROR:viz_main_impl.cc(152)] Exiting GPU process due to errors during initialization
ERROR: ld.so: object '/usr/$LIB/libgamemodeauto.so.0' from LD_PRELOAD cannot be preloaded (cannot open shared object file): ignored.
ERROR: ld.so: object '/usr/$LIB/libgamemodeauto.so.0' from LD_PRELOAD cannot be preloaded (cannot open shared object file): ignored.
[0324/174023.627:ERROR:dxva_video_decode_accelerator_win.cc(1405)] DXVAVDA fatal error: could not LoadLibrary: msvproc.dll: Module not found. (0x7E)
ERROR: ld.so: object '/usr/$LIB/libgamemodeauto.so.0' from LD_PRELOAD cannot be preloaded (cannot open shared object file): ignored.
ERROR: ld.so: object '/usr/$LIB/libgamemodeauto.so.0' from LD_PRELOAD cannot be preloaded (cannot open shared object file): ignored.
ERROR: ld.so: object '/usr/$LIB/libgamemodeauto.so.0' from LD_PRELOAD cannot be preloaded (cannot open shared object file): ignored.

```

System Info
```
[System]
OS:              Zorin OS 16 focal
Arch:            x86_64
Kernel:          5.13.0-27-generic
Desktop:         zorin:GNOME
Display Server:  x11

[CPU]
Vendor:          AuthenticAMD
Model:           AMD Ryzen 7 5800X 8-Core Processor
Physical cores:  8
Logical cores:   16

[Memory]
RAM:             15.5 GB
Swap:            2.0 GB

[Graphics]
Vendor:          NVIDIA Corporation
OpenGL Renderer: NVIDIA GeForce RTX 3080/PCIe/SSE2
OpenGL Version:  4.6.0 NVIDIA 470.86
OpenGL Core:     4.6.0 NVIDIA 470.86
OpenGL ES:       OpenGL ES 3.2 NVIDIA 470.86
Vulkan:          Supported


```

**Comment by MTPeyton** [comment]
I ultimately solved the problem and was able to launch D4. I nuked Zorin OS and installed Arch. This also meant that I now had the latest nvidia drivers. I then installed Lutris through Flatpak, setup a valid directory (that it had permissions to via Flatseal) to store the game data. Then I used the Lutris install script for Battle.net. It installed and launched, and then allowed me to launch Diablo 4, without any additional configuration. I didn't even need the latest GE Proton... the default Wine version in Lutris worked fine.
